spec/support/with_stubbed_stdout.rb

```diff
@@ -14,7 +14,7 @@ def with_stubbed_stdout(stdin: nil, stderr: nil, &block)
     { stdout: nil, stderr: nil }
   else
     stderr ||= ''
-    stdout = run(stdin:, stderr:, &block)
+    stdout = run(stdin:, stderr:, &)
```

Comment:
How do you feel about this `&` syntax? It took me a minute to parse

Reply:
14 years writing &block, nooooooo 😆 . I hope I get used to it. Yes, let's leave as it is and see.

Reply:
The docs are built using a different ruby version  < 3